fe/fe-core/src/main/java/com/starrocks/connector/ConnectorMetadata.java

```diff
@@ -0,0 +1,37 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.connector;
+
+import com.google.common.collect.Lists;
+import com.starrocks.catalog.Table;
+
+import java.util.List;
+
+public interface ConnectorMetadata {
```

Comment:
it should call Metadata, because 'LocalMetastore' need implement it.

Reply:
Metadata is a general name, besides ConnectorMetadata, there are going to have few `-Metadata`, so we need a more specific name. Local table is going to be integrated to Connector in the future, so `LocalMetastore` is ok to derived from `ConnectorMetadata`

Reply:
agree, but It still sounds a little strange😓